src/sdltiles.cpp

```diff
@@ -395,6 +395,8 @@ bool WinCreate()
         }
     }
     if( software_renderer ) {
+        if( get_option<bool>( "FRAMEBUFFER_ACCEL" ) )
+            SDL_SetHint( SDL_HINT_FRAMEBUFFER_ACCELERATION, "1" );
```

Comment:
This isn't effectively detailed in CODE_STYLE.md. Is there a more detailed outline?

Same line braced, or K&R style for one liners like this?